editors/README.md

```diff
@@ -0,0 +1,35 @@
+# Text Editor Configuration
+
+Validate a swarm-lang file using: `swarm format ./file.sw`
+
+Swarm comes with a language server protocol (LSP) server.
+Make sure the `swarm` program is present in your PATH, then
+configure your editor.
+
+## EMACS
+
+The current LSP implementation features the following extensions:
+
+* error diagnostic on load/save
+
+Using [lsp-mode](https://github.com/emacs-lsp/lsp-mode):
+
+Load the [swarm-mode.el](../contribs/swarm-mode.el) and start
+the `M-x lsp` service in a *swarm-mode* buffer.
+
+## VS Code and VS Codium
+
+The [swarm-lang](./vscode) extension provides highlighting and an
+LSP client. That is if you have `swarm` executable in PATH, then
+the executable will be used as LSP server to show errors as you type.
+
+You can get it by:
+- installing from MS marketplace ([link](XXX))
```

Comment:
Can this be filled in now?

Reply:
Yes, I forgot to add the link :slightly_smiling_face: